packages/SwingSet/misc-tools/parse_timestamps.py

```diff
@@ -96,12 +99,13 @@ def __init__(self, vatID, cranknum, vd, when, monotime):
         self.rx_delivery = None
         self.tx_result = None
         self.rx_result = None
+        self.computrons = None
         self.vd = vd
         if vd[0] == "message":
             target = vd[1]
             msg = vd[2]
             methargsCD = msg["methargs"]
-            method = json.loads(methargsCD["body"])[0]
+            method = json.loads(methargsCD["body"][1:])[0]
```

Comment:
```suggestion
            method = json.loads(methargsCD["body"].removeprefix("#"))[0]
```
Or actually, many of these files do `json.loads(…["body"])` that would be more robust and more clear as `unmarshal(…["body"])` as in extract-non-empty-blocks.py.
```suggestion
            method = unmarshal(methargsCD["body"])[0]
```